ui/src/app/lib/atlasmap-data-mapper/models/transition.model.ts

```diff
@@ -124,7 +124,9 @@ export class FieldActionConfig {
 
       // Check for matching types - date.
       if (this.matchesDate(this.sourceType, selectedSourceField.type)) {
-        if ((this.multipleTransformations(fieldPair)) || (this.matchesDate(this.targetType, selectedTargetField.type))) {
+        if ((this.multipleTransformations(fieldPair)) || (this.matchesDate(this.targetType, selectedTargetField.type)) ||
+            ((this.targetType === 'INTEGER') && ((this.name === 'DayOfWeek') || (this.name === 'DayOfMonth')) ||
```

Comment:
weren't they fit with the rule? Since these transformations have different type between source and target, it only needs to appear as source transformation where source field type is Date/Time.

Reply:
yes - they don't need to call out the functions any more - will fix